syft/execution/state.py

```diff
@@ -71,13 +71,7 @@ def read(self):
                     parent_plan.state.state_placeholders.append(placeholder)
                     parent_plan.role.placeholders[placeholder.child.id] = placeholder
 
-        tensors = []
-        for placeholder in self.state_placeholders:
-            # State elements from sub plan should not be reported when read() is used
-            if "#inner" not in placeholder.tags:
-                tensor = placeholder.child
-                tensors.append(tensor)
-        return tensors
+        return [ph for ph in self.state_placeholders if "#inner" not in ph.tags]
```

Comment:
Wonder if `#inner` could become a (derived?) property instead of a tag. 🤔

Reply:
Sure! I didn't want to change that right now but I'd be glad to do it if everybody agrees 👍

Reply:
Yeah, not at all required for this PR, but figured it followed in the spirit of the other tag removal you did.